tests/h/groups/views_test.py

```diff
@@ -544,6 +562,61 @@ def test_leave_publishes_leave_event(Group, session_model):
     })
 
 
+def annotation(document, groupid=u'__world__', shared=False):
+    """
+    Add an annotation of the given Document to the db and return it.
+
+    The given Document must already have an associated DocumentURI.
+
+    """
+    annotation = models.Annotation(
+        userid=u'fred', groupid=groupid, shared=shared,
+        target_uri=document.document_uris[0].uri)
+    db.Session.add(annotation)
+    return annotation
+
+
+def document(db, uri):
```

Comment:
Do you think we should have a `db_session` fixture like we do for the API tests?

I think we'd end up with `db_session` as a fixture and `annotation()` as a helper function though, rather than both as fixtures or both as helper functions.

`annotation()` is a function because it's often called more than once by a single test, passing different values to it each time, sometimes those values are documents produced by earlier calls to `document()` in the same test function ... I think it's just easier to do this as a helper function. But for something like a `db_session` that can easily be done as a fixture I'd do so.